convert.py

```diff
@@ -43,6 +43,9 @@ def convert_onnx_to_trt(onnx_weights, image_shape, nc, auto_install=True):
     parser.add_argument('--no-auto-install',
                         action='store_true',
                         help="Turn off auto install feature")
+    parser.add_argument('--fp16',
+                        action='store_true',
```

Comment:
I think default behavior should be ```fp16``` since it's going to be more common. What do you think?

Reply:
[similar suggestion](https://github.com/BlueMirrors/cvu/pull/22#discussion_r799150626) as in pull request https://github.com/BlueMirrors/cvu/pull/22#discussion_r799150626